src/include/distributed/master_protocol.h

```diff
@@ -56,6 +56,9 @@
 	"SELECT worker_apply_shard_ddl_command (" UINT64_FORMAT ", %s, %s)"
 #define WORKER_APPLY_SHARD_DDL_COMMAND_WITHOUT_SCHEMA \
 	"SELECT worker_apply_shard_ddl_command (" UINT64_FORMAT ", %s)"
+#define WORKER_APPLY_INTER_SHARD_DDL_COMMAND \
+	"SELECT worker_apply_inter_shard_ddl_command (" UINT64_FORMAT ", " UINT64_FORMAT \
```

Comment:
how does worker_apply_inter_shard_ddl_command ensure that we're using the right schema?